docs/known_issues.md

```diff
@@ -14,10 +14,23 @@ As a workaround, use `tag.turbo_frame` instead of `turbo_frame_tag`.
 
 Note: For the same functionality as `turbo_frame_tag(my_model)`, use `tag.turbo_frame(id: dom_id(my_model))`.
 
-## form_for compatibility
+## `form_for` compatibility
 
 ViewComponent [isn't compatible](https://github.com/viewcomponent/view_component/issues/241) with `form_for` helpers by default.
 
+This means that it isn't supported to pass a form object to a view component.
```

Comment:
Actually it can work for some simple cases: you pass the form builder reference (usually `f`) to the component, as an argument, and use it in the component, ie. `f.text_field :name`. But for more complex cases (passing blocks to form helpers) you may have duplicate or wrongly ordered content.

Reply:
You are right, @Spone. I am uncertain if we need to mention this details here... My focus is more the "way forward" with the options one has...

Reply:
If there is demand, I can of course elaborate on this. @Spone and @ianterrell, what do you think?

Reply:
@ianterrell, thanks for giving some insights. What is your argument to not put the label, description and error component into a form builder? For example, viewcomponent-form gem would allow you to combine both worlds.

Reply:
Do you mean this [view_component-form](https://github.com/pantographe/view_component-form) gem? For large projects I try to run as few dependencies as possible. We don't even use Devise. I wonder if we should be using ViewComponent 😅.

I don't have a strong argument not to put the label etc into a form builder, other than it just seemed natural to use view components directly, and we have some inertia in that direction — we use them almost everywhere else, after all. I didn't think about using form builders at first, and now we have a lot of forms built this way. 

I was surprised when in some cases ViewComponents didn't work. In fact, in some cases they did work and then stopped working, when the global buffer changes were being tried out. 

All that said... since they play so poorly together — or at least, poorly together _sometimes_ — we probably should transition to using our own form builder.

Reply:
Overall, I think if we're trying to document how things currently are, we should aim at being as specific as possible, so that people can make an informed choice between the available options.

I would draw the line at describing specifics about external libraries, since I prefer people get their info first-hand from their docs (and we don't have to keep it updated on our side).